gcp/website/frontend3/src/linter.js

```diff
@@ -357,21 +357,43 @@ document.addEventListener("DOMContentLoaded", function () {
 
     tabSwitch.appendChild(tabButton);
 
+    const vulnJsonId = `vuln-json-${bugId}`;
+    const findingsJsonId = `findings-json-${bugId}`;
+
     const tabContent = document.createElement("div");
     tabContent.className = "tab-content";
     tabContent.id = tabId;
-    tabContent.innerHTML = `
```

Comment:
Hmm..this is much less clear, keeping innerHTML is fine here, we just need to sanitise bugId